vulcan/formula.py

```diff
@@ -75,14 +77,18 @@ def install(self, force=False):
                 shutil.rmtree(destination)
 
             # Move formula into place.
-            origin = os.path.join(tmpdir, self.origin_name)
-            shutil.move(origin, destination)
+            log.info('Moving formula %s to %s' %(self.name, destination))
+            shutil.move(tmpdir, destination)
```

Comment:
This won't work correctly. It'll sync out foo-formula to a temp directory and then move the entire thing to the destination.

What I want to happen is sync out foo-formula and move foo-formula/foo to the destination.